RealmSwift/SwiftUI.swift

```diff
@@ -134,7 +136,8 @@ private final class ObservableStoragePublisher<ObjectType>: Publisher where Obje
         subscribers.append(AnySubscriber(subscriber))
         if value.realm != nil && !value.isInvalidated, let value = value.thaw() {
             // if the value is managed
-            let token =  value._observe(subscriber)
+            let token = value._observe(subscriber, keyPaths: keyPaths)
+//            let token = value._observe(subscriber)
```

Comment:
Can be removed?

Reply:
Resolved in [b8e9ee0](https://github.com/realm/realm-cocoa/pull/7249/commits/b8e9ee02afed35f06f77bd688886ad6b7aea627d)